bpf/tc/attach.go

```diff
@@ -217,6 +234,7 @@ func (ap AttachPoint) listAttachedPrograms() ([]attachedProg, error) {
 			p := attachedProg{
 				pref:   sm[1],
 				handle: sm[2],
+				object: strings.Split(sm[3], ":")[0],
```

Comment:
I'm a bit wary of relying on extracting this from the output.  ISTR that with Sridhar's libbpf work, these values were changing or getting truncated.